timescaledb/overview/limitations.md

```diff
@@ -14,51 +14,49 @@ hypertables.
-  continuous aggregates are downsamples of the data, this does not usually
-  create a problem.
-* Reordering chunks is not supported.
-* Tablespaces cannot be attached to a distributed hypertable on the
-  access node. It is still possible to attach tablespaces on data nodes.
-* Roles and permissions are assumed to be consistent across the nodes
-  of a distributed database, but consistency is not enforced.
-* Joins on data nodes are not supported. Joining a distributed
-  hypertable with another table requires the other table to reside on
-  the access node. This also limits the performance of joins on
-  distributed hypertables.
-* Tables referenced by foreign key constraints in a distributed
-  hypertable must be present on the access node and all data
-  nodes. This applies also to referenced values.
-* Parallel-aware scans and appends are not supported.
-* A consistent restore point for backup/restore across nodes is not
-  natively provided; care must be taken when restoring individual
-  backups to access and data nodes.
-* For native replication limitations, see the
-  [native replication section][native-replication].
-* User defined functions have to be manually installed on the data nodes
-  so that the function definition is available on both access and data
-  nodes. This is particularly relevant for functions that are
-  registered with `set_integer_now_func`.
+*   Distributed scheduling of background jobs is not supported. Background jobs
+    created on an access node are scheduled and executed on this access node
+    without distributing the jobs to data nodes.
+*   Continuous aggregates can aggregate data distributed across data nodes, but
+    the continuous aggregate itself must live on the access node. This could
+    create a limitation on how far you can scale your installation, but because
+    continuous aggregates are downsamples of the data, this does not usually
+    create a problem.
+*   Reordering chunks is not supported.
+*   Tablespaces cannot be attached to a distributed hypertable on the access
+    node. It is still possible to attach tablespaces on data nodes.
+*   Roles and permissions are assumed to be consistent across the nodes of a
+    distributed database, but consistency is not enforced.
+*   Joins on data nodes are not supported. Joining a distributed hypertable with
+    another table requires the other table to reside on the access node. This
+    also limits the performance of joins on distributed hypertables.
+*   Tables referenced by foreign key constraints in a distributed hypertable
+    must be present on the access node and all data nodes. This applies also to
+    referenced values.
+*   Parallel-aware scans and appends are not supported.
+*   Distributed hypertables do not natively provide a consistent restore point
+    for backup and restore across nodes. Use the
+    `create_distributed_restore_point` command, and make sure you take care when
```

Comment:
```suggestion
    [`create_distributed_restore_point`][create_distributed_restore_point] command, and make sure you take care when
```

Makes it a tiny bit easier on the reader to include the link, I think. Link is here: https://docs.timescale.com/api/latest/distributed-hypertables/create_distributed_restore_point/ but can't add it in via GH review because it's too far from the edited portion.